docs/how-to/jupyter-notebook.rst

```diff
@@ -0,0 +1,379 @@
+.. _manage_notebooks:
+
+Manage Jupyter Notebooks
+========================
+
+This guide describes how to manage `Jupyter Notebooks`_ within your Data Science Stack (DSS) environment.
+
+All actions can be performed using the DSS Command Line Interface (CLI). 
+
+Create a notebook
+-----------------
+
+You can create a Jupyter Notebook using the DSS CLI.
+This notebook includes different packages and toolkits depending on the image used to create it.
+
+1. **Select an image**:
+
+    Before creating a notebook, you need to select an image that includes the packages and toolkits you need.  
+    To see a list of recommended images and their aliases, do:
+
+    .. code-block:: bash
+
+        dss create --help
+
+    The output includes a list of recommended images and their aliases.
+    For example, this guide uses the image `kubeflownotebookswg/jupyter-scipy:v1.8.0`
```

Comment:
Same here for indentation, should we have a uniform story for lists that have a lot of text and code blocks?